packages/ckeditor5-html-embed/src/htmlembedediting.js

```diff
@@ -166,6 +166,7 @@ export default class HtmlEmbedEditing extends Plugin {
 						// it's enough to update the model – the entire widget will be reconverted.
 						if ( newValue !== state.getRawHtmlValue() ) {
 							editor.execute( 'updateHtmlEmbed', newValue );
+							editor.focus();
```

Comment:
I think that the a8410bf was on point here.

As per [this comment](https://github.com/ckeditor/ckeditor5/issues/714#issuecomment-721086621) in the PR that introduced `editor.focus()` the semantics of both is a bit different, even though it's a shortcut for the `editor.editing.view.focus()`.

IIRC - the reason behind this is that `editor.focus()` might change in the future. A similar thing is being reverted - buttons should use `editor.editing.view.focus()`.